src/main/java/com/google/dicomwebfuse/dao/FuseDaoHelper.java

```diff
@@ -80,13 +83,22 @@
     }
 
     if (studyList.size() > MAX_STUDIES_IN_DICOM_STORE) {
-      throw new ToManyResultsException("Too large DICOM Store. In " + dicomPath.getDicomStoreId() +
+      LOGGER.warn("Too large DICOM Store. In " + dicomPath.getDicomStoreId() +
           " over " + MAX_STUDIES_IN_DICOM_STORE + " Studies!");
     }
 
     return studyList;
   }
 
+  public static Study getStudy(FuseDao fuseDao, CloudConf cloudConf,
```

Comment:
getSingleStudy() for consistency with getSingleSeries()?

Reply:
done